.github/workflows/_test.yml

```diff
@@ -5,24 +5,54 @@ on:
 permissions: {}
 
 jobs:
+  lint-test:
+    name: Lint Test
+    strategy:
+      matrix:
+        python-version: [3.x]
+
+    env:
+      TOXENV: lint
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout TUF
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          cache-dependency-path: 'requirements*.txt'
+
+      - name: Install dependencies
+        run: |
+          python3 -m pip install --constraint requirements-build.txt tox coveralls
+
+      - name: Run tox (${{ env.TOXENV }})
+        # See TOXENV environment variable for the testenv to be executed here
+        run: tox
+
   tests:
     name: Tests
+    needs: lint-test
+    continue-on-error: ${{ matrix.failure-allowed }}
```

Comment:
I don't think you need continue-on-error anymore, or do you?

Reply:
Yes, that is to keep the job running if anyone of them fails.